docs/source/apidocs/wavefunction.rst

```diff
@@ -1,8 +1,11 @@
+:orphan:
+
 Wavefunction Methods
 ====================
 
 .. currentmodule:: pyquil.api
 .. autoclass:: pyquil.api.WavefunctionSimulator
+    :noindex:
```

Comment:
Motivating warning:

```
WARNING: duplicate object description of pyquil.api.WavefunctionSimulator, other instance in apidocs/autogen/pyquil.api.WavefunctionSimulator, use :noindex: for one of them
```

This warning occurs because there is another reference to `pyquil.api.WavefunctionSimulator` in an `autosummary` section in `docs/source/apidocs/simulators.rst`, which causes the `autodoc` extension to generate a competing "stub" summary for this class in `docs/source/autogen/pyquil.api.WavefunctionSimulator.rst`. Sphinx then gets confused and winds up linking to one or the other at various places in the docs.

Currently, for example, clicking the `WavefunctionSimulator` link near the top of the [simulators page](http://docs.rigetti.com/en/stable/apidocs/simulators.html) takes you to [this `autoclass` definition](http://docs.rigetti.com/en/stable/apidocs/wavefunction.html#pyquil.api.WavefunctionSimulator), whereas clicking on the [`WavefunctionSimulator` link in the index](http://docs.rigetti.com/en/stable/genindex.html#W) takes you to the [`autodoc`-generated stub](http://docs.rigetti.com/en/stable/apidocs/autogen/pyquil.api.WavefunctionSimulator.html#pyquil.api.WavefunctionSimulator).

One could argue that adding a `:noindex:` here is a bit silly, since the page is also `:orphaned:` (see my other comment), which means that the page won't appear in any `toctree` and neither will any cross-ref links point to this `WavefunctionSimulator` `autoclass` template, so maybe it should just be deleted.  Or maybe it shouldn't be orphaned.  Or maybe this page should be linked to directly from `simulators.rst` rather than being included in the `autosummary` table there.... Or maybe there is another solution that I'm not seeing.  I'm open to suggestions from any passing sphinx-wizards.